recipes/agouti/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set name = "AGouTI" %}
+{% set version = "0.1.35" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://test-files.pythonhosted.org/packages/c3/47/0c05eec9090085f0f398e07e26e16cfaa116bca0e11b701ee16bd8fe50e2/{{ name }}-{{ version }}.tar.gz
```

Comment:
the test-files URL looks wronf

Reply:
thank you for the review. I will do my best to resolve the problem